src/src/AdminBundle/Controller/EventController.php

```diff
@@ -2,20 +2,178 @@
 
 namespace AdminBundle\Controller;
 
+use AdminBundle\Entity\Event;
+use AdminBundle\Entity\EventOrganizers;
+use AdminBundle\Entity\EventType;
+use AdminBundle\Entity\Organizer;
+use AdminBundle\Form\EventForm;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
 
 class EventController extends Controller
 {
-    //todo
+    /**
+     * @Route("/event_type/edit/{id}/event/add", name="event_add")
+     * @Method({"GET", "POST"})
+     */
+    public function createAction($id, Request $request)
+    {
+        $repository = $this->getDoctrine()->getRepository(EventType::class);
+        $eventType = $repository->findOneBy(['id' => intval($id)]);
+
+        $newEvent = new Event();
+        $newEvent->setEventTypeId($eventType);
+        $form = $this->createForm(EventForm::class, $newEvent);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($newEvent);
+            $em->flush();
+
+            return $this->redirectToRoute('event_type_edit', ['id' => $id]);
+        }
+
+        return $this->render('AdminBundle:Event:create.html.twig', array(
+            'form' => $form->createView(),
+        ));
+    }
+
+    private static $modalInputOrganizers = 'assignOrganizer';
+
+    /**
+     * @Route("event_type/edit/{id}/event/{event_id}/edit", name="event_edit")
+     * @Method({"GET", "POST"})
+     */
+    public function editAction($id, $event_id, Request $request)
+    {
+        $em = $this->getDoctrine();
+        $eventOrganizers = $em->getRepository(EventOrganizers::class)->findBy(['eventId' => $event_id]);
+
+        $eventOrganizersIds = [];
+        foreach ($eventOrganizers as $eventOrganizer) {
+            $eventOrganizersIds[] = $eventOrganizer->getOrganizerId();
+        }
+
+        $organizers = $em->getRepository(Organizer::class)->findById($eventOrganizersIds);
+
+        $repository = $this->getDoctrine()->getRepository(Event::class);
+        $event = $repository->findOneBy(['id' => intval($event_id)]);
+
+        $form = $this->createForm(EventForm::class, $event);
+
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $em = $this->getDoctrine()->getManager();
+            $em->persist($event);
+            $em->flush();
+
+            return $this->redirectToRoute('event_type_edit', ['id' => $id]);
+        }
+
+        return $this->render('AdminBundle:Event:edit.html.twig', array(
+            'form' => $form->createView(),
+            'organizers' => $organizers,
+            'event_type_id' => $id,
+            'modal_input_organizers' => self::$modalInputOrganizers,
```

Comment:
preco? aky problem toto riesi? podla mna ziaden, iba to komplikuje veci. ci si len nechcela mat rovnaky kus stringu na viacerych miestach?

Reply:
moj zlaty, tento string posielame do modalu, kde je to ako `name` a `id` toho inputu. 
a v modali potom vykoname get akciu. 
potom tento string pouzijeme v `assignOrganizerAction` ako kluc aby sme vytiahli hodnotu z requestu.

Reply:
Tak znova.

`self::$modalInputOrganizers` vytiahla si to do triedy kvoli tomu, nech nemas string ktory to obsahuje, na viacerych miestach v kode?

Reply:
ano